dataset: hellaswag	No info from HuggingFace
dataset: hellaswag	splits: train: 39905 rows, test: 10003 rows, val: 10042 rows
dataset: hellaswag	split: train	columns: ind, activity_label, ctx_a, ctx_b, ctx, endings, source_id, split, split_type, label
***ERROR***
model: None
data: H:hellaswag
eqn: None
splits: train
split: train
task: None
error: (<class 'requests.exceptions.HTTPError'>, HTTPError('405 Client Error: Not Allowed for url: https://huggingface.co/api/models-tags-by-type'), <traceback object at 0x7fab1202b440>)
